packages/x-scheduler/src/joy/agenda-view/AgendaView.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+import * as React from 'react';
+import clsx from 'clsx';
+import { getAdapter } from '../../primitives/utils/adapter/getAdapter';
+import { SchedulerValidDate } from '../../primitives/models';
+import { TimeGrid } from '../../primitives/time-grid';
+import { TimeGridEvent } from '../event/time-grid-event/TimeGridEvent';
+import { AgendaViewProps } from './AgendaView.types';
+import { CalendarEvent } from '../models/events';
+import { isWeekend } from '../internals/utils/date-utils';
+import { useTranslations } from '../internals/utils/TranslationsContext';
+import './AgendaView.css';
+import { useDayList } from '@mui/x-scheduler/primitives/use-day-list';
+import { AgendaEvent } from '../event/agenda-event/AgendaEvent';
+
+const adapter = getAdapter();
+
+export const AgendaView = React.forwardRef(function AgendaView(
+  props: AgendaViewProps,
+  forwardedRef: React.ForwardedRef<HTMLDivElement>,
+) {
+  const getDayList = useDayList();
+
+  const today = adapter.date('2025-05-26');
```

Comment:
```suggestion
  const store = useEventCalendarStore();
  const visibleDate = useSelector(store, selectors.visibleDate);

    
  const days = React.useMemo(
    () => getDayList({ date: visibleDate.startOf('week'), amount: 12 }),
    [getDayList, today],
  );            
```

Why 12 by the way? :laughing:

Reply:
No particular reason. Arbitrary number that felt right for proving the concept of the agenda view 🙈 We can think about making it customizable from the settings at some point 🤔